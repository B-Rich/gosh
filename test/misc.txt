              Document Title

               Norman Feske


A paragraph without a chapter.

Abstract
########

Ohh.. .wie abstrakt....

Chapter One
###########

Some _accentuations with spaces in-between_

A paragraph with an
'apostroph' at the beginning of the line.



