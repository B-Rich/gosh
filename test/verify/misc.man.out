.TH "misc.txt" 7 "" "TU Dresden" "DROPS Manual"
.SH NAME
misc.txt \- Document Title
A paragraph without a chapter.
.SH "ABSTRACT"
Ohh.. .wie abstrakt....
.SH "CHAPTER ONE"
Some 
.I accentuations with spaces in-between

A paragraph with an
\(cqapostroph' at the beginning of the line.
.SH AUTHOR
Norman Feske
